firmware/config/engines/mazda_miata_vvt.cpp

```diff
@@ -362,10 +362,12 @@ static void setCommonMazdaNB() {
 	set4EC_AT();
 }
 
+#pragma GCC diagnostic ignored "-Wunused-function"
```

Comment:
that's a hack not real solution

real solution would be to move static method closer to real usages so that it is conditionally compiled together with usages

Reply:
+1, we should never be turning on/off warnings unless there's a really good reason to, and this isn't one